artemis-jms-client/src/main/java/org/apache/activemq/artemis/jms/client/compatible1X/ActiveMQTextCompabileMessage.java

```diff
@@ -51,4 +52,10 @@ public ActiveMQTextCompabileMessage(ClientMessage message, ClientSession session
    public ActiveMQTextCompabileMessage(TextMessage foreign, ClientSession session) throws JMSException {
       super(foreign, session);
    }
+
+
+   @Override
+   protected SimpleString checkPrefix(SimpleString address) {
+      return ActiveMQCompatibleMessage.checkPrefix1X(address);
```

Comment:
Should this be on all compatible1x messages

Reply:
It is.. that's what ActiveMQTextCompatibleMessage is

Reply:
ah.. this one was missing. .it was a typo